plugins/vendor/lib/model/zoom/RetrieveDataFromZoom.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * @package plugins.venodr
+ * @subpackage model.zoom
+ */
+class RetrieveDataFromZoom
```

Comment:
In my opinion, a better name would be zoomWrapper and this class should contain any methods that interact with the zoom API.